Documentation/ceph-quickstart.md

```diff
@@ -31,7 +31,7 @@ If you're feeling lucky, a simple Rook cluster can be created with the following
 
 ```console
 git clone --single-branch --branch {{ branchName }} https://github.com/rook/rook.git
-cd cluster/examples/kubernetes/ceph
```

Comment:
There are a number of places in the docs where this same issue would also apply (looks like 8 places where we have a `git clone` command). Can you also update those at the same time?

Reply:
Adding the others that I can find, and ammending original commit.